### ISSUE TITLE
Font Size problems

### ISSUE THREAD

**Issue by sggts04** [sggts04/dankcli#1]
Sometimes font size is too big in long height but small width photos and is too small in **very** small height but long width photos and

**Comment by dheiberg** [comment]
Got any examples?

**Comment by sggts04** [comment]
@dhdavvie Example Image I used:
![](https://cdn.discordapp.com/attachments/483978508716998656/555474034570297375/das.PNG)

typed command:
```
python -m dankcli "das.png" "see the font here is like very small because the height is so small"
```

and it produced this image with unreadable small text:
![](https://i.imgur.com/B6hR5RW.png)

**Comment by sggts04** [comment]
I need a better way to calculate font size, currently I am just dividing the photo height by 13, thus this problem.

**Comment by dheiberg** [comment]
One way I see of solving this is by having a lower bound on font size. So you could do `return max(math.floor(imgSize[1]/13), 12)` in [getFontSize](https://github.com/sggts04/dankcli/blob/master/dankcli/functions.py#L8) so that if there is a small height, it'll have a minimum font sizde of 12 (or whatever is suitable). Simple but should do the trick

**Comment by sggts04** [comment]
should do the trick, but what about the photos with large height but small width, kinda like a phone screenshot. the font-size would be too big there, so an upper bound for that maybe?

**Comment by sggts04** [comment]
Should be finally fixed in v0.5.6
